src/ioc/index.ts

```diff
@@ -1,12 +1,18 @@
 import IoCContainer from 'ioc-lite';
 
-// import { Logger } from '../services/logger';
-// import { HTTP } from '../services/http';
-// import { Users } from '../services/users';
+import { Logger } from '../services/logger';
+import { HTTP } from '../services/http';
+import { Users } from '../services/users';
 
 export const createIoCContainer = () =>  {
   const ioc = new IoCContainer();
-  // you can register some resources right now below...
+
+  ioc.registerClass('logger', Logger);
+  ioc.registerClass('users', Users);
+  ioc.registerClass('http', HTTP)
+
+  const config = (window as any).__CONFIG__;
+  ioc.register('config', config.api);
```

Comment:
Why do you write this code here ?
You can do right the same things in the src/index.ts file (inside app function)